src/main/java/de/htwberlin/GradesOverview.java

```diff
@@ -3,9 +3,13 @@
 public class GradesOverview {
 
   private int[] grades;
+  private int index=0;
+  private double gradesSum;
+  private double avarage;
+  private String outputString;
 
   public GradesOverview() {
-    grades = new int[5];
+    grades = new int[]{0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
hier ginge auch einfach

```suggestion
    grades = new int[10];
```

0 ist der Default-Wert in einem Array vom Datentyp `int`.